packages/examples-site/src/App.tsx

```diff
@@ -19,8 +20,147 @@ import PageFiltersDropdowns from "./pages/Filters/FiltersDropdownsPage/FiltersDr
 import PageFiltersAdvanced from "./pages/Filters/FiltersAdvPage/FiltersAdvancedPage";
 import PageFiltersAdvancedAdditive from "./pages/Filters/FiltersAdvancedAdditivePage/FiltersAdvancedAdditivePage";
 import DashboardLayout from "./pages/DashboardLayoutPage";
+import Sidebar from "../../patterns/src/components/sidebar/sidebar";
+import { SidebarProvider } from "../../patterns/src/components/sidebar/sidebar-context";
+import { ReactRouterProviderAdapter } from "./adapters/react-router-adapter";
+import { menuItems } from "./config/menu-items";
+import { BiLogoGithub } from "react-icons/bi";
+
 export const alertsManager = createAlertsManager();
 
+const LayoutContainer = styled.div`
+  display: grid;
+  height: 100vh;
+  width: 100%;
+`;
+
+const InnerContainer = styled.div`
+  min-height: 100vh;
+  overflow: hidden;
+`;
+
+const MainContent = styled.main`
+  display: grid;
+  height: 100%;
+  grid-template-columns: max-content 1fr;
+`;
+
+const ContentArea = styled.div`
+  height: 100%;
+  overflow: auto;
+
+  &:last-child::after {
+    height: 3rem;
+    content: " ";
+    display: block;
+  }
+`;
```

Comment:
Instead of appending additional height, do you see any side effects by calculating it?

```suggestion
const ContentArea = styled.div`
  height: calc(100vh - 3.5rem);
  overflow: auto;
`;
```

Reply:
That works great! 👏 Just committed that change.